samples/bluetooth/hci_rpmsg/sample.yaml

```diff
@@ -16,6 +16,7 @@ tests:
     tags: bluetooth
     integration_platforms:
       - nrf5340dk_nrf5340_cpunet
+      - nrf5340_audio_dk_nrf5340_cpunet
```

Comment:
Add `nrf5340_audio_dk_nrf5340_cpunet` to next test too that build hci_rpmsg for iso_receive configuration.

Reply:
Missed that one, thanks. Also added to standard hci_rpmsg.